### PR TITLE
Fix has_Pending()

### PR DESCRIPTION
**Pull request by adibas03**

### CODE REVIEW

**Comment by lsaether** [comment]
For example, we need to not allow someone to send a 1 wei gas price transaction to all transaction request contracts, then the `hasPending` check would fail currently.

**Comment by lsaether** [comment]
Will test this using CLI before merge.

File: src/Actions/Pending.ts [hunk 1]
```diff
@@ -15,13 +21,18 @@ const _hasPendingParity = async (conf: any, txRequest: any) => {
...
       },
-      (err: Error, res: any) => {
+      async (err: Error, res: any) => {
         if (err) reject(err);
 
         const hasTx =
           res &&
           res.result &&
-          !!res.result.filter((tx: any) => tx.to === txRequest.address).length;
+          !!await res.result.filter(async (tx: any) => {
```
Comment (lsaether) [inline src/Actions/Pending.ts]: While testing with CLI I discovered that this block returns `True` when it should return `False`

**Comment by lsaether** [comment]
@Bagaric The thing about unit tests with this is that you can't get these to work on `ganache-cli`, they require a Partity node and live transactions to test. I think we can merge this since I'm doing QA testing with CLI (once that checks out).

However I'm not saying we shouldn't have tests for these. We should consider having tests that will run on Kovan and we can run those once a week for example... Or before release.

**Comment by lsaether** [comment]
Closes #56

**Comment by Bagaric** [comment]
@lsaether Ok, I'm ok with merging these without unit tests if that's the case.

**Pull request merged**